Src/CMakeLists.txt

```diff
@@ -32,9 +32,6 @@ if (CMAKE_Fortran_COMPILER_LOADED)
       INTERFACE_INCLUDE_DIRECTORIES
       $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/mod_files>
       )
-   # adding those is a work-around that avoids that downstream C++ projects have
-   # to `enable_language(Fortran)`
-   target_link_libraries(amrex PUBLIC ${CMAKE_Fortran_IMPLICIT_LINK_LIBRARIES})
```

Comment:
Note: this served us reasonably well but did not cover all bases, e.g. we saw issues on macOS with Fortran functionality, OpenMP and LLVM+GFortran mixes as well as Windows' MSVC + CUDA #1585